includes/admin/functions.php

```diff
@@ -194,3 +194,28 @@ function amp_bootstrap_admin() {
 	$site_health = new SiteHealth();
 	$site_health->init();
 }
+
+/**
+ * Whether to activate the new onboarding feature.
+ *
+ * @return bool
+ */
+function amp_should_use_new_onboarding() {
+	global $wp_version;
+
+	if ( version_compare( $wp_version, '5.0', '<' ) ) {
+		return false;
+	}
+
+	// @todo Remove this check when the onboarding feature is released.
+	if ( '1' === filter_input( INPUT_GET, 'amp-new-onboarding', FILTER_SANITIZE_NUMBER_INT ) ) {
```

Comment:
Note that I generally don't recommend using `filter_input()` because (1) core doesn't use it, and (2) as far as I know the input values can't be mutated during unit testing.

Reply:
Granted this is just a feature flag, so no big deal.